tests/firebird/test-firebird.cpp

```diff
@@ -516,6 +516,16 @@ TEST_CASE("Firebird blobs", "[firebird][blob]")
     sql.commit();
 
     sql.begin();
+    {
+        // Read from default-constructed BLOB
+        blob b(sql);
+
+        CHECK(b.get_len() == 0);
+
+        char buf[5];
+        std::size_t read = b.read_from_start(buf, 5);
+        CHECK(read == 0);
+    }
```

Comment:
I suppose all Blob-related test cases can be removed as they should be handled via common-tests